test/test_pub_sub.py.in

```diff
@@ -9,11 +9,14 @@ import unittest
 
 import launch
```

Comment:
We can drop this now that you're importing ExecuteProcess and OpaqueFunction below

```suggestion
```